src/redux/books/books.js

```diff
@@ -0,0 +1,23 @@
+import { createReducer, createAction } from '@reduxjs/toolkit';
+
+// Define action type for addding & removing
+export const ADD_BOOK = createAction('books/add');
+export const REMOVE_BOOK = createAction('books/remove');
```

Comment:
- [ ] I will advise you to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:
- const ADD_BOOK = bookstore/books/ADD_BOOK
- const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
This is the conventional and the popular approach in the industry.

Reply:
Thank you for your good explanation - I have implemented your suggestions.